physcraper/__init__.py

```diff
@@ -3142,6 +2931,7 @@ def sp_dict(self, downtorank=None):
                                 tax_id = item.split(":")[1]
                             # tax_id = self.ids.ottid_to_ncbiid(ott_id)
                 if self.downtorank is not None:
+                    #TODO this is trying to get the ncbi_id from a name? or...?
```

Comment:
No, it uses the taxon_id to get the id of the corresponding rank defined in you input file.

Reply:
Ah ok - so set self.downtorank = 'genus', it would add the sequence to the sp_dict using the id for the genus? 
e.g. sp_dict = {Homo_tax_id:[Homo sapiens otu'], 'Pan_tax_id':['Pan troglodytes otu', 'Pan paniscus otu']},  
whereas, if downtorank = None the sp_dict would look more like 
{Homo_sapiens_tax_id:['Homo sapiens otu1', 'Homo sapiens otu2'], Pan_troglodytes_tax_id:['Pan troglodytes otu'] .. etc}?

 Is that correct?

Reply:
yes